docs/bigquery-cdcTarget.md

```diff
@@ -65,4 +65,10 @@ Staging tables names are generated by prepending this prefix to the target table
 datasets when a drop table or drop database event is encountered. When set to true, the replicator will
 not delete a table or dataset. Instead, it will fail and retry until the table or dataset does not exist.
 If the dataset or table does not already exist, no manual intervention is required. The event will be
-skipped as normal.
\ No newline at end of file
+skipped as normal.
+
+**Dataset Name**: Name of the dataset to be created in the BQ query. It's optional and by default the dataset
```

Comment:
minor nit: `created in the BigQuery.` instead of `created in the BQ query.`

Reply:
good catch :)